bus/bus.go

```diff
@@ -654,7 +653,7 @@ func New(s Syncer, cm ChainManager, tp TransactionPool, w Wallet, hdb HostDB, cs
 		"POST   /wallet/prepare/renew": b.walletPrepareRenewHandler,
 		"GET    /wallet/pending":       b.walletPendingHandler,
 
-		"GET    /hosts":          b.hostsHandler,
+		"POST   /hosts":          b.hostsHandlerPOST,
```

Comment:
this is unfortunate but I decided to propose it, alternative would/could be if the whitelist lives in the stores

Reply:
Yet another argument for `POST /rpc` 😂